Source/Fuse.Drawing.Primitives/Circle.uno

```diff
@@ -96,8 +96,9 @@ namespace Fuse.Drawing.Primitives
 				VertexCount: _bufferIndex.Count();
 				
 				float2 VertexPosition: V0 * (radius + extend*2);
+				float2 VertexPositionScaled: VertexPosition / radius;
```

Comment:
What if radius is 0 ?

Reply:
Good point. Perhaps earlying out in such cases would be a good move.